packages/next/src/shared/lib/normalized-asset-prefix.ts

```diff
@@ -0,0 +1,16 @@
+export function normalizedAssetPrefix(assetPrefix: string | undefined): string {
+  const escapedAssetPrefix = assetPrefix?.replace(/^\/+/, '') || false
+
+  // assetPrefix as a url
+  if (escapedAssetPrefix && escapedAssetPrefix.startsWith('http')) {
+    return escapedAssetPrefix.split('://', 2)[1]
+  }
+
+  // assetPrefix is set to `undefined` or '/'
+  if (!escapedAssetPrefix || escapedAssetPrefix === '') {
```

Comment:
```suggestion
  if (!escapedAssetPrefix) {
```
`!escapedAssetPrefix` already checks for `''` as it's `falsy`